react/components/Modal/README.md

```diff
@@ -1,386 +1,480 @@
+
+  return (
+    <>
+      <Button onClick={onOpen}>Open modal</Button>
+
+      <Modal
+        isOpen={isOpen}
+        title="Ready to start?"
+        aria-describedby="modal-description"
+        responsiveFullScreen
+        bottomBar={
+          <div className="nowrap">
+            <span className="mr4">
+              <Button variation="tertiary" onClick={onClose}>
+                Cancel
+              </Button>
+            </span>
+            <span>
+              <Button variation="primary" onClick={onClose}>
+                Send
+              </Button>
+            </span>
+          </div>
+        }
+        onClose={onClose}
+      >
+        <div className="flex flex-column flex-row-ns">
+          <div className="w-100 w-50-ns">
+            <p className="f3 f1-ns fw3 gray" id="modal-description">
+              Discover our unified commerce platform and see your business
+              thrive.
+            </p>
+          </div>
+          <div className="w-100 w-50-ns mv4 pv6-ns pl6-ns">
+            <div className="w-100 mv6">
+              <Input placeholder="Name" size="large" />
             </div>
-          }
-        onClose={this.handleCloseModal}>
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent
-            semper eget magna sit amet maximus. In rutrum, justo sodales euismod
-            dignissim, justo orci venenatis lectus, vel semper turpis nunc a
-            justo. Pellentesque tempor sodales ante, a dignissim metus posuere
-            sed. Aenean dapibus nisl non finibus ultrices. Sed in urna nisi.
-            Fusce malesuada, diam eget accumsan pharetra, diam nisl cursus
-            mauris, facilisis pretium risus elit in diam. Duis tincidunt
-            interdum purus sit amet interdum. Phasellus non elit id lacus
-            euismod vehicula nec at velit. Ut ullamcorper pharetra vulputate.
-            Praesent volutpat sem et dolor facilisis molestie in et velit.
-            Quisque suscipit augue vitae nunc euismod, lacinia volutpat lacus
-            porta. Curabitur eget enim urna. Sed neque lectus, pulvinar eget
-            vulputate ut, tristique nec leo.
-          </p>
+            <div className="w-100 mv6">
+              <Input placeholder="Last name" size="large" />
+            </div>
+            <div className="w-100 mv6">
+              <Input placeholder="Corporate email" size="large" />
+            </div>
+            <div className="w-100 mv6">
+              <Input placeholder="Company" size="large" />
+            </div>
+            <div className="w-100 mv6">
+              <Input placeholder="Annual revenue" size="large" />
+            </div>
+            <div className="w-100 mv6">
+              <Input placeholder="Do you have e-commerce?" size="large" />
+            </div>
+          </div>
+        </div>
+      </Modal>
+    </>
+  )
+}
+
+;<ModalExample />
+```
+
+#### Sizes
```

Comment:
I think using different size buttons doesn't make clear the difference between modal sizes. I would use the same structure as in [Alert](https://styleguide.vtex.com/#/Components/Notification/Alert) scenarios, splitting each **size** alternative, and giving more context (size, usage examples) for each one.

Reply:
Thanks for the suggestion. I separated and improved the examples for each size, can you check?

Reply:
https://styleguide-git-feat-new-modal-with-typescript.styleguide-core.now.sh/#/Components/Overlays/Modal

Reply:
Awesome!